Packs/Ipstack/Integrations/Ipstack/Ipstack_test.py

```diff
@@ -23,20 +31,30 @@ def test_right_location_format(mocker, requests_mock):
         and contain DBotScore calculations.
     """
     mocker.patch.object(demisto, 'params', return_value={'proxy': 'proxy',
-                                                         'apikey': 'apikey',
+                                                         'credentials': {'password': 'apikey'},
```

Comment:
```suggestion
                                                         'credentials': {'password': 'password'},
```
Use a different value for different fields.